src/Plugin.NFC/iOS/NFC.iOS.cs

```diff
@@ -205,6 +207,8 @@ public override void DidDetectTags(NFCTagReaderSession session, INFCTag[] tags)
 						// let the user decide if ndef support is needed
 						OnMessageReceived?.Invoke(nTag);
 
+						session.AlertMessage = Configuration.Messages.NFCSuccessRead;
```

Comment:
Is this need or was this just for debugging?

Reply:
This is necessary for sub task FPC-1106